src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -322,6 +380,30 @@ public SignUpRequest signUp(String email, String password, String connection) {
      * @return a Request to configure and execute.
      */
     public AuthRequest login(String emailOrUsername, String password) {
+        return this.login(emailOrUsername, password != null ? password.toCharArray() : null);
+    }
+
+    /**
+     * Creates a log in request using the 'Password' grant and the given credentials.
+     * i.e.:
+     * <pre>
+     * {@code
+     * AuthAPI auth = new AuthAPI("me.auth0.com", "B3c6RYhk1v9SbIJcRIOwu62gIUGsnze", "2679NfkaBn62e6w5E8zNEzjr-yWfkaBne");
+     * try {
+     *      TokenHolder result = auth.login("me@auth0.com", new char[]{'s','e','c','r','e','t})
+     *          .setScope("openid email nickname")
+     *          .execute();
+     * } catch (Auth0Exception e) {
+     *      //Something happened
+     * }
+     * }
+     * </pre>
+     *
+     * @param emailOrUsername the identity of the user.
+     * @param password        the password of the user.
+     * @return a Request to configure and execute.
+     */
+    public AuthRequest login(String emailOrUsername, char[] password) {
```

Comment:
do we want to keep these connection-less methods around? They would use the "default directory" value for the tenant. But it might be the case that specifying null for the connection param (on the other methods) would have the same effect (unchecked).

Reply:
We may want to deprecate some APIs where we have overloaded methods that we can simplify. This should come in a separate PR.